src/components/AppSidebar/AppSidebar.vue

```diff
@@ -16,7 +16,7 @@
       <template v-for="tab in tabs">
         <v-tab
           :key="tab.name"
-          :to="tab.page"
+          :to="`/${appConfig}${tab.page}`"
```

Comment:
I'd move this to a computed prop.

Reply:
Can you explain why? we are in a list here, so `tab.page` is dynamic. If you want to move it into a computed prop, you have to construct a list of urls alongside the tabs. the other option is to make a method which accepts `tab.page` and returns a value. 

But I think this is just clean as it is.